### PR TITLE
Shadow Style

### PR DESCRIPTION
**Pull request by ssorallen**
Move styles inline to work in shadow DOM
- Remove tag name selector where scoping is no longer needed
- Replace tag name selector with `:host` when selecting based on host
  class name

Also...
- Add content to `<button>` elements to prevent them from collapsing in
  Firefox

### CODE REVIEW

File: core-toolbar.html [hunk 1]
```diff
@@ -59,19 +59,114 @@
...
+
+    /*
+     * make elements (e.g. buttons) respond to mouse/touch events
+     *
+     * `.toolbar-tools` disables touch events so multiple toolbars can stack and not
+     * absorb events. All children must have pointer events re-enabled to work as
+     * expected.
+     */
+    .toolbar-tools > *:not([disabled]),
+    ::content > *:not([disabled]) {
```
Comment (morethanreal) [inline core-toolbar.html]: Is `toolbar-tools` a class set in the local or light children? It looks like `.toolbar-tools > *:not([disabled])` won't match anything.

File: index.html [hunk 2]
```diff
@@ -8,15 +8,15 @@
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
 -->
 <html>
-  <head>
+<head>
 
-    <script src="../webcomponentsjs/webcomponents.js"></script>
-    <link rel="import" href="../core-component-page/core-component-page.html">
+  <script src="../webcomponentsjs/webcomponents.js"></script>
```
Comment (morethanreal) [inline index.html]: `webcomponents-lite.js` since we don't use the shadow DOM polyfill?

File: core-toolbar.html (hunk 1 above)
Comment (ssorallen) [inline core-toolbar.html]: Aha thanks for that. This is the point you mentioned yesterday about wrapping `<content>` in another container.

File: index.html (hunk 2 above)
Comment (ssorallen) [inline index.html]: :+1:

**Comment by ssorallen** [comment]
@morethanreal, can you take another look?

**Comment by morethanreal** [comment]
lgtm

**Comment by ssorallen** [comment]
Thanks. :bee:

**Pull request merged**